official/NormeDiProgetto/res/sections/primaryProcesses.tex

```diff
@@ -236,6 +288,38 @@ \section{Processi primari}
 		  \item L’\glossaryItem{efficacia} dei metodi di analisi statica \`e strettamente correlata alla qualit\`a di strutturazione del codice.
                 \end{itemize}
 
+		\paragraph*{Documentazione nel codice}
```

Comment:
@mdipirro dove avevi aggiunto queste cose? secondo me si possono ripetere in documenti diversi